public/js/site/main.js

```diff
@@ -93,4 +95,16 @@ console.log(error.message);
     // ...
     console.log(errorMessage);
   });
+  $("#logout").on("click", function (e) {
+
+  firebase.auth().signOut().then(function() {
+    // Sign-out successful.
+    console.log("login success");
```

Comment:
"login success" -> "successfully logged out"
I know it's just logging message and not user impact issue
but I still think it's important to show correct log messages for other developers